src/main/g8/src/main/scala/$package$/api/Api.scala

```diff
@@ -6,9 +6,9 @@ import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.server.{Directives, Route}
 import akka.http.interop._
 import akka.http.scaladsl.model.StatusCodes.NoContent
-import play.api.libs.json.JsObject
 import zio._
 import zio.logging._
+import de.heikoseeberger.akkahttpziojson.ZioJsonSupport
```

Comment:
It would be better to stick to original solution - make JsonSupport extend ZioJsonSupport, so that we don't need to import ZioJsonSupport everywhere.

Reply:
@jczuchnowski sure I'll do that.
Thanks!